tests/test_response_parser.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for the GraphQL Response Parser.
+
+These tests are worthless until I have a schema I can work with.
```

Comment:
check in the starwars schema (augmented if necessary with some custom type definitions) and use it for testing, rather than depending on Money or whatever.